src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -76,11 +79,17 @@
       <Context>Project</Context>
     </PropertyPageSchema>
 
+    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)ProjectDebugger.xaml">
```

Comment:
Mostly unrelated to this change, but does the ProjectDebugger.xaml file need to get added to `src\VsixV3\ProjectSystemPackage\Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swr`?